vertx-web-client/src/main/java/io/vertx/ext/web/client/CacheOptions.java

```diff
@@ -0,0 +1,63 @@
+package io.vertx.ext.web.client;
+
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.codegen.annotations.Fluent;
+import io.vertx.ext.web.client.impl.CacheInterceptor;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static io.vertx.ext.web.client.impl.CacheInterceptor.CacheKeyValue.*;
+import static io.vertx.ext.web.client.impl.CacheInterceptor.CacheKeyValue.CONTENT_TYPE;
+import static io.vertx.ext.web.client.impl.CacheInterceptor.CacheKeyValue.PARAMS;
+
+/**
+ * @author Alexey Soshin
+ */
+@DataObject
+public class CacheOptions {
```

Comment:
I think this class needs at least one `JsonObject` constructor otherwise the annotation processor will not compile it